toqito/helper/bell_npa_constraints.py

```diff
@@ -0,0 +1,159 @@
+"""Generates NPA hierarchy constraints for Bell inequalities."""
+
+from typing import List, Tuple, Union
+
+import cvxpy
+import numpy as np
+
+from toqito.helper.npa_hierarchy import Symbol, _gen_words, _reduce
+
+
+def _word_to_p_cg_index(word: Tuple[Symbol, ...], oa: int, ob: int, ma: int, mb: int) -> Union[int, None]:
+    """Map an operator word to its corresponding index in the flattened CG vector."""
+    dim_a = (oa - 1) * ma
+    dim_b = (ob - 1) * mb
+    row_dim = dim_a + 1
+    col_dim = dim_b + 1
+    order = "F"
+    if not word:
+        return 0
+    if len(word) == 1:
+        s = word[0]
+        if s.player == "Alice":
+            row_idx = (oa - 1) * s.question + s.answer + 1
+            return np.ravel_multi_index((row_idx, 0), (row_dim, col_dim), order=order)
+        if s.player == "Bob":
+            col_idx = (ob - 1) * s.question + s.answer + 1
+            return np.ravel_multi_index((0, col_idx), (row_dim, col_dim), order=order)
+    if len(word) == 2:
+        s_a, s_b = word
+        if s_a.player == "Alice" and s_b.player == "Bob":
+            row_idx = (oa - 1) * s_a.question + s_a.answer + 1
+            col_idx = (ob - 1) * s_b.question + s_b.answer + 1
+            return np.ravel_multi_index((row_idx, col_idx), (row_dim, col_dim), order=order)
+    return None
```

Comment:
Python functions will (by default) return `None`, so we can likely remove this line:
```suggestion
```